app/javascript/packages/document-capture/components/form-steps.tsx

```diff
@@ -9,135 +10,197 @@ import useForceRender from '../hooks/use-force-render';
 
-/**
- * @typedef FormStepComponentProps
- *
- * @prop {(nextValues:Partial<V>)=>void} onChange Update values, merging with existing values.
- * @prop {OnErrorCallback} onError Trigger a field error.
- * @prop {Partial<V>} value Current values.
- * @prop {FormStepError<V>[]} errors Current active errors.
- * @prop {FormStepError<V>[]} unknownFieldErrors Current top-level errors.
- * @prop {RegisterFieldCallback} registerField Registers field by given name, returning ref
- * assignment function.
- *
- * @template V
- */
+export type OnErrorCallback = (error: Error, options?: { field?: string | null }) => void;
 
-/**
- * @typedef FormStep
- *
- * @prop {string} name Step name, used in history parameter.
- * @prop {import('react').FC<FormStepComponentProps<Record<string,any>>>} form Step form component.
- * @prop {(object)=>boolean=} validator Optional function to validate values for the step
- */
+export interface FormStepComponentProps<V> {
+  /**
+   * Update values, merging with existing values.
+   */
+  onChange: (nextValues: Partial<V>) => void;
 
-/**
- * @typedef FieldsRefEntry
- *
- * @prop {import('react').RefCallback<HTMLElement>} refCallback Ref callback.
- * @prop {boolean} isRequired Whether field is required.
- * @prop {HTMLElement?} element Element assigned by ref callback.
- */
+  /**
+   * Trigger a field error.
+   */
+  onError: OnErrorCallback;
 
-/**
- * @typedef FormStepsProps
- *
- * @prop {FormStep[]=} steps Form steps.
- * @prop {Record<string,any>=} initialValues Form values to populate initial state.
- * @prop {FormStepError<Record<string,Error>>[]=} initialActiveErrors Errors to initialize state.
- * @prop {boolean=} autoFocus Whether to automatically focus heading on mount.
- * @prop {(values:Record<string,any>)=>void=} onComplete Form completion callback.
- * @prop {()=>void=} onStepChange Callback triggered on step change.
- */
+  /**
+   * Current values.
+   */
+  value: Partial<V>;
 
-/**
- * @typedef FormStepsContext
- *
- * @prop {boolean} isLastStep Whether the current step is the last step in the flow.
- * @prop {boolean} canContinueToNextStep Whether the user can proceed to the next step.
- * @prop {() => void} onPageTransition Callback invoked when content is reset in a page transition.
- */
+  /**
+   * Current active errors.
+   */
+  errors: FormStepError<V>[];
+
+  /**
+   * Current top-level errors.
+   */
+  unknownFieldErrors: FormStepError<V>[];
 
-export const FormStepsContext = createContext(
-  /** @type {FormStepsContext} */ ({
-    isLastStep: true,
-    canContinueToNextStep: true,
-    onPageTransition: () => {},
-  }),
-);
+  /**
+   * Registers field by given name, returning ref assignment function.
+   */
+  registerField: RegisterFieldCallback;
+}
+
+export interface FormStep {
+  /**
+   * Step name, used in history parameter.
+   */
+  name: string;
+
+  /**
+   * Step form component.
+   */
+  form: FC<FormStepComponentProps<Record<string, any>>>;
```

Comment:
Where did `FC` come from?

Reply:
It's imported above:

https://github.com/18F/identity-idp/blob/55bef8e559b46e1ed5aff2b0cb5e0d33208e081b/app/javascript/packages/document-capture/components/form-steps.tsx#L2

Short-hand for `FunctionComponent<Props>`

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/77cb04b6ffd8469c8f3b3c60229cf2ff0ce7680e/types/react/index.d.ts#L548